fuse_examples/classification/mnist/runner.py

```diff
@@ -66,11 +66,19 @@
 ##########################################
 # Train Common Params
 ##########################################
+TRAIN_COMMON_PARAMS = {}
+# ============
+# Model
+# ============
+TRAIN_COMMON_PARAMS['model'] = 'lenet' # 'resnet18' or 'lenet'
+
 # ============
 # Data
 # ============
-TRAIN_COMMON_PARAMS = {}
-TRAIN_COMMON_PARAMS['data.batch_size'] = 30
+if TRAIN_COMMON_PARAMS['model'] == 'lenet':
```

Comment:
I think that in both cases you can set to 100. 
Unless you checked and got out of memory error.